shanoir-ng-front/src/app/studies/study/study.component.html

```diff
@@ -421,7 +421,7 @@ <h2 class="header command-zone" i18n="Create study|Title@@studyDetailCreateTitle
 					(delete) = "delete()"
 					(cancel)="goToView()"
 					(back)="goBack()">
-				<dataset-download  *ngIf="mode == 'view' && hasDownloadRights()" [studyId]="id" [datasetIds]="selectedDatasetIds"></dataset-download>
```

Comment:
We can't download "selected datasets" anymore ?

Reply:
oups